openwisp_controller/subnet_division/signals.py

```diff
@@ -0,0 +1,3 @@
+from django.dispatch import Signal
+
+subnet_ips_provisioned = Signal(providing_args=['instance', 'ip_obj'])
```

Comment:
I'd call this something like `post_provisioning` or `subnet_provisioned`. I would use simply `ip_address` or `ip` (instead of `ip_obj`).

Please add it to the README.